action-server/alembic/versions/2c2215d3e953_initial_db_setup.py

```diff
@@ -0,0 +1,94 @@
+"""Initial db setup.
+
+Revision ID: 2c2215d3e953
+Revises:
+Create Date: 2020-04-21 13:32:44.422928
+
+"""
+import structlog
+from sqlalchemy import Boolean, Column, DateTime, func
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.sql.schema import CheckConstraint, ForeignKey
+from sqlalchemy.types import Integer, String
+
+from alembic import op
+
+log = structlog.get_logger(__name__)
+
+revision = "2c2215d3e953"
```

Comment:
How is this guarantee that the migration are run in order?

Reply:
The migrations have a `down_revision` attributes which defines the migration sequence. However I will change them to a sequential number to make the sequence obvious by looking at filenames.